xml/System.Diagnostics.CodeAnalysis/FeatureGuardAttribute.xml

```diff
@@ -29,14 +29,35 @@
   </Attributes>
   <Docs>
     <summary>
-      <para>Indicates that the specified public static boolean get-only property</para>
-      <para>guards access to the specified feature.</para>
+      <para>
+        Indicates that the specified public static boolean get-only property
+        guards access to the specified feature.</para>
     </summary>
     <remarks>
-      <para>Analyzers can use this to prevent warnings on calls to code that is</para>
-      <para>annotated as requiring that feature, when the callsite is guarded by a</para>
-      <para>call to the property.</para>
+      <para>
+        Analyzers can use this to prevent warnings on calls to code that is
+        annotated as requiring that feature, when the callsite is guarded by a
+        call to the property.</para>
     </remarks>
+    <example>
+      <code language="csharp">
+          if (Feature.IsSupported)
+              Feature.Implementation();
+
+          public class Feature
+          {
+              [FeatureGuard(typeof(RequiresDynamicCodeAttribute))]
+              internal static bool IsSupported => RuntimeFeature.IsDynamicCodeSupported;
+
+              [RequiresDynamicCode("Feature requires dynamic code support.")]
+              internal static Implementation() => ...; // Uses dynamic code
+          }
+      </code>
+      When the app is built with `&lt;PublishAot&gt;true&lt;PublishAot&gt;`, the call to
```

Comment:
This isn't rendering as angle brackets, probably because it's code-fenced. See https://review.learn.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.featureguardattribute?view=net-9.0&branch=pr-en-us-10304#examples.